### PR TITLE
Add possibility to update jsonContent through upate_porperties_in_ass…

### PR DESCRIPTION
**Pull request by PaulKili**
…ets()

- [x] I opened a merge request on `kili` on a branch with the same name than the branch of the current pull request and forced security tests

### CODE REVIEW

File: kili/mutations/asset/__init__.py [hunk 1]
```diff
@@ -137,7 +137,8 @@ def update_properties_in_assets(self, asset_ids: List[str],
                                     external_ids: Optional[List[str]] = None,
                                     priorities: Optional[List[int]] = None, json_metadatas: Optional[List[dict]] = None, consensus_marks: Optional[List[float]] = None,
                                     honeypot_marks: Optional[List[float]] = None, to_be_labeled_by_array: Optional[List[List[str]]] = None, contents: Optional[List[str]] = None,
-                                    status_array: Optional[List[str]] = None, is_used_for_consensus_array: Optional[List[bool]] = None, is_honeypot_array: Optional[List[bool]] = None):
+                                    jsonContents: Optional[List[str]] = None, status_array: Optional[List[str]] = None, is_used_for_consensus_array: Optional[List[bool]] = None,
```
Comment (marcenacp) [inline kili/mutations/asset/__init__.py:140]: Camel case

File: kili/mutations/asset/__init__.py [hunk 2]
```diff
@@ -230,14 +235,16 @@ def update_properties_in_assets(self, asset_ids: List[str],
         for i, properties in enumerate(zip(*data)):
             for property, property_value in zip(property_names, properties):
                 data_array[i][property] = property_value
-        for i in range(nb_assets_to_modify):
+        for i in range(nb_assets_to_modify):
```
Comment (marcenacp) [inline kili/mutations/asset/__init__.py:238]: Formatting

File: kili/mutations/asset/__init__.py [hunk 3]
```diff
@@ -230,14 +235,16 @@ def update_properties_in_assets(self, asset_ids: List[str],
         for i, properties in enumerate(zip(*data)):
             for property, property_value in zip(property_names, properties):
                 data_array[i][property] = property_value
-        for i in range(nb_assets_to_modify):
+        for i in range(nb_assets_to_modify):            
             data_array[i]['shouldResetToBeLabeledBy'] = should_reset_to_be_labeled_by_array[i]
+            print(data_array)
```
Comment (marcenacp) [inline kili/mutations/asset/__init__.py:240]: print

File: kili/mutations/asset/__init__.py [hunk 4]
```diff
@@ -230,14 +235,16 @@ def update_properties_in_assets(self, asset_ids: List[str],
...
         variables = {
             'whereArray': where_array,
             'dataArray': data_array
         }
         result = self.auth.client.execute(
             GQL_UPDATE_PROPERTIES_IN_ASSETS, variables)
+        print(result)
```
Comment (marcenacp) [inline kili/mutations/asset/__init__.py:247]: print

**Review by kili-technology — APPROVED** [review]

**Review by marcenacp — APPROVED** [review]

**Pull request merged**